fleet_management_api/script_args/args.py

```diff
@@ -106,11 +107,15 @@ def _parse_arguments(parser: argparse.ArgumentParser) -> ScriptArgs:
         if test is not None:
             config_dict["database"]["test"] = test
         config = _APIConfig(**config_dict)
+        _update_config_with_args(args, config)
+        return ScriptArgs(args, config)
+
+    except ConfigFileNotFound as e:
+        logging.error(f"Configuration file not found. {e}")
+        raise
     except Exception as e:
-        print(f"Check the configuration file ('{config_path}'). {e}")
+        logging.error(f"Check the configuration file ('{config_path}'). {e}")
```

Comment:
It is common to print errors during initializing application to stdout. Does this work, since logger is properly initialized with values from config?

Reply:
The log is handled by the root logger. However, for verbose=false (when logs are not printed to the console), the print statement has been added to the fleet_management_api.__main__